test/integration.test.ts

```diff
@@ -531,4 +531,26 @@ describe('integration tests', () => {
     expect(createdPlanes).toHaveLength(250);
     expect(queriedPlanes.records).toHaveLength(PAGINATION_DEFAULT_SIZE);
   });
+
+  test('multiple errors in one response', async () => {
+    // @ts-expect-error
+    const invalidUsers: User[] = [{ full_name: 1 }, { full_name: 2 }];
+
+    expect(client.db.users.createMany(invalidUsers)).rejects.toMatchInlineSnapshot(`
```

Comment:
Is this missing an await? 🤔

Reply:
Nope, jest .rejects already takes care of the promise